mediaqueries/Overview.bs

```diff
@@ -1783,6 +1783,52 @@ Rare Interaction Capabilities: the 'any-pointer' and 'any-hover' features</h3>
 		or to hide them altogether to reduce visual clutter if ''(any-pointer: fine)'' is false.
 	</div>
 
+
+
+<!--
+  ██     ██  ██████  ████████ ████████  ████████  ████████  ████████ ████████  ██████
+  ██     ██ ██    ██ ██       ██     ██ ██     ██ ██     ██ ██       ██       ██    ██
+  ██     ██ ██       ██       ██     ██ ██     ██ ██     ██ ██       ██       ██
+  ██     ██  ██████  ██████   ████████  ████████  ████████  ██████   ██████    ██████
+  ██     ██       ██ ██       ██   ██   ██        ██   ██   ██       ██             ██
+  ██     ██ ██    ██ ██       ██    ██  ██        ██    ██  ██       ██       ██    ██
+   ███████   ██████  ████████ ██     ██ ██        ██     ██ ████████ ██        ██████
+-->
+
+<h2 id='mf-user-preferences'>
+User Preference Media Features</h2>
+
+<h3 id="prefers-reduced-motion">
+Detecting the desire for less motion on the page: the 'prefers-reduced-motion' feature</h3>
+
+	<pre class='descdef mq'>
+	Name: prefers-reduced-motion
+	Value: default | reduce
+	For: @media
+	Type: discrete
+	</pre>
+
+	The 'prefers-reduced-motion' media feature is used to detect if the user
+	has requested the system minimize the amount of animation or motion it
+	uses.
+
+	<dl dfn-type=value dfn-for="@media/prefers-reduced-motion">
+		<dt><dfn>default</dfn>
+		<dd>
+			Indicates that the user has made no preference known
+			to the system. This keyword value evaluates as false
+			in the <a>boolean context</a>.
+
+		<dt><dfn>reduce</dfn>
+		<dd>
+			Indicates that user has notified the system that they
+			prefer an interface that minimizes the amount of movement
+			or animation, preferably to the point where all non-essential
+			movement is removed.
+
+	</dl>
+
+
```

Comment:
I'd rather not do that, at least not without a summary of what you want people to go look for in that giant rambling of ours.